generator.yaml

```diff
@@ -499,6 +532,8 @@ resources:
         from:
           operation: DescribeModelPackage
           path: ModelApprovalStatus
+        late_initialize:
```

Comment:
why do we need late initialize for this field?

Reply:
Refer to custom_delta for model package this field is not mandatory but is however set to a default value of 	`// Default is for ModelApprovalStatus to be set to pending manual approval`

Reply:
let's remove this field from status since its part of the spec already